dist/linux/systemd.service.in

```diff
@@ -1,9 +1,12 @@
 [Unit]
 Description=@NAME@
 After=graphical-session.target
+After=dbus.socket
+Requires=dbus.socket
 
```

Comment:
These are needed because they were implied by `Type=dbus`. This ensures that D-Bus is ready before we start up.

Reply:
Done.